modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -459,4 +459,92 @@ definition {
 		}
 	}
 
+	@description = "LPS-171625 - Verify that after updating a custom object entry, triggered user notifications using Object Author and Current User terms are filled correctly for the To and Subject fields"
+	@priority = 5
+	test CanUseObjectAuthorAndCurrentUserTermsInUserNotificationOnCustomObject {
+		property custom.properties = "feature.flag.LPS-171625=true";
+		property portal.acceptance = "true";
+
+		task ("Given a custom object and a entry") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 180339",
+				objectName = "CustomObject180339",
+				pluralLabelName = "Custom Objects 180339");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject180339");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject180339");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customField",
+				objectName = "CustomObject180339",
+				value = "Entry Test");
+		}
+
+		task ("And given a user notification template with the applicable fields filled with the Author and Current User terms") {
+			UserNotificationAPI.createNotificationTemplate(
+				en_US = "[%CURRENT_USER_EMAIL_ADDRESS%] [%CURRENT_USER_FIRST_NAME%] \n[%CUSTOMOBJECT180339_AUTHOR_LAST_NAME%] [%CUSTOMOBJECT180339_CUSTOMFIELD%]",
```

Comment:
This line break is really necessary? If not, I think it's better we don't use to avoid future problems on CI or flakes